tensorrt_llm/_torch/models/modeling_llama.py

```diff
@@ -472,17 +472,15 @@ def forward(
                 hidden_states_activated_experts = hidden_states[1]
                 num_activated_experts_per_node = hidden_states[2]
                 experts_to_token_score = hidden_states[3]
-                activated_expert_global_ids = hidden_states[4]
-                hidden_states, residual = self.deepseek_allreduce(
-                    hidden_states_activated_experts,  # not used
-                    [
-                        residual, self.next_layer_layernorm.weight,
-                        num_activated_experts_per_node, experts_to_token_score,
-                        hidden_states_activated_experts, shared_output,
-                        activated_expert_global_ids
-                    ],
-                    self.next_layer_layernorm.variance_epsilon,
-                    AllReduceFusionOp.MOE_ALLREDUCE_RESIDUAL_RMS_NORM,
+
+                hidden_states, residual = self.moe_allreduce(
+                    residual,
+                    self.next_layer_layernorm.weight,
+                    device_num_experts=num_activated_experts_per_node,
+                    scale_input=experts_to_token_score,
+                    active_experts_token_input=hidden_states_activated_experts,
+                    token_input=shared_output,
+                    eps=self.next_layer_layernorm.variance_epsilon,
```

Comment:
FYI, this will not get exercised in CI because fusion is turned off in the model.